extensions/amp-audio/amp-audio.md

```diff
@@ -78,6 +78,10 @@ If present, the audio will automatically loop back to the start upon reaching th
 
 If present, will mute the audio by default.
 
+##### controlsList
+
+If its value is set to `nodownload`, the download button will be hidden.
```

Comment:
Please update to: 

Same as [controlsList](https://developer.mozilla.org/en-US/docs/Web/API/HTMLMediaElement/controlsList) attribute of HTML5 `audio` element. Only supported by certain browsers. Please see https://developer.mozilla.org/en-US/docs/Web/API/HTMLMediaElement/controlsList for details.